src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,341 @@
+class Guides::Frontend::Internationalization < GuideAction
+  guide_route "/frontend/internationalization"
+
+  def self.title
+    "Internationalization - Handle Multiple Languages"
+  end
+
+  def markdown : String
+    <<-MD
+    This describes how to handle USERS with multiple language preferences.  At somepoint we can document how to add dynamic switching from the frontend too.
+
+    ## **Step 1** - add i18n shard to `shard.yml`
+
+    ```
+    dependencies:
+      i18n:
+        github: vladfaust/i18n.cr
+        version: ~> 0.1.1
+    ```
+    and of course install the shard
+    ```
+    shards install
+    ```
+
+    ## **Step 2** - add a starter localization and anothers needed:
+
+    first make a locales folder:
+    ```
+    mkdir config/locales
+    ```
+
+    then add a localization
+    ```
+    # config/locales/en.yml
+    en:
+      auth:
+        sign_in: "Sign-In"
+        sign_up: "Sign-up"
+        update_pwd: "Update Password"
+        reset_pwd_request: "Reset Password"
+      default:
+        page_name: Welcome
+      me:
+        email: Mail
+        profile: Profil
+        next: "Next, you may want to"
+        auth_guides: "Check out the authentication guides"
+        modify_page: "Modify this page"
+        after_signin: "Change where you go after sign in"
+    ```
+
+    ## **Step 3** - configure i18n to work with Lucky
+    ```
+    # config/i18n.cr
+    require "i18n"
+    require "i18n/backends/yaml"
```

Comment:
I think these `requires` should actually go in Step 1 and should be required in `src/shards.cr`.

Reply:
cool learning a lot - thanks